test/unit/MessageCreationUtil.test.js

```diff
@@ -506,7 +506,7 @@ describe('MessageCreationUtil', () => {
                 requestId,
             })
 
-            expect(streamMessage.getStreamId()).toBe('destinationAddress'.toLowerCase()) // sending to subscriber's inbox stream
+            expect(streamMessage.getStreamId()).toBe('destinationAddress') // sending to subscriber's inbox stream
```

Comment:
Should the `streamId` actually be lowercase?

Reply:
`system/keyexchange` vs `SYSTEM/keyexchange`

Reply:
Ah yes, of course. I think this expectation here was a remnant of the key exchange stream id being just the address. The mixed-case problem hasn't gone anywhere though, so both `0xabc` and `0xABC` should both result in the key exchange messages getting sent to `SYSTEM/keyexchange/0xabc`, never `SYSTEM/keyexchange/0xABC`. I wonder if there's test coverage for this elsewhere?